crates/protocol/registry/src/chain_list.rs

```diff
@@ -1,14 +1,52 @@
 //! List of OP Stack chains.
 
 use alloc::{string::String, vec::Vec};
+use alloy_chains::Chain as AlloyChain;
 
 /// List of Chains.
 #[derive(Debug, Clone, Default, Hash, Eq, PartialEq, serde::Serialize, serde::Deserialize)]
+#[serde(transparent)]
 pub struct ChainList {
     /// List of Chains.
     pub chains: Vec<Chain>,
 }
```

Comment:
Can use [`derive_more::IntoIterator`](https://docs.rs/derive_more/latest/derive_more/derive.IntoIterator.html) here instead of manual impl.